src/commands/dev/lib/builder-cache.ts

```diff
@@ -82,7 +81,14 @@ export async function cleanCacheDir(output: Output): Promise<void> {
 /**
  * Install a list of builders to the cache directory.
  */
-export async function installBuilders(packages: string[]): Promise<void> {
+export async function installBuilders(
+  packages: string[],
+  update: boolean = false
```

Comment:
What's the use case for `update === false`?

Reply:
It's not implemented yet, but in the `updateBuildMatches()` function it should invoke `installBuilder()` if any matches changed, in case the builder has not yet been installed. I was thinking it would do `update: false` in that case during runtime.